CONTRIBUTING.md

```diff
@@ -34,13 +34,17 @@ First of all, thank you for contributing to Meilisearch! The goal of this docume
 
 To run this project, you will need:
 
-- Node >= 12
+- Node >= 14 && node <= 17
 - Npm >= v7
 - A google account
 - Latest version of `firebase-tools` the Firebase CLI:
-``` bash
-npm install -g firebase-tools
-```
+  ``` bash
+  yarn global add firebase-tools
+  ```
+  Add the directory for the commands of the packages installed globally in yarn, to access of firebase binary:
```

Comment:
What do you mean `in yarn` ?
Is this sentence meant to be for the code above?

Reply:
No, it's for the code below. Actually, if you install `firebase-tools` with `npm` you just have to run `npm install -g firebase-tools` but with `yarn` I had some trouble and I had to run the other command to make it works:
```sh
  export PATH="$(yarn global bin):$PATH"
```